src/main/resources/application-local.yml

```diff
@@ -66,6 +67,6 @@ user-allocations:
         wales: JIMSNOWLDAP
         nat: JIMSNOWLDAP
     reallocated-assessments:
```

Comment:
i can't find this referenced anywhere, so i think it can be removed (reallocated-assessments)?